README.md

```diff
@@ -46,24 +48,29 @@ The `generate` subcommand is the core of vog. It allows you to generate both imm
 Example call: `./vendor/bin/vog generate ./value.json`
 
 ## Configuration
-You can create a vog_config.php file in your project to modify the behaviour of vog. 
-Default settings generate PSR2 compliant value objects for PHP 7.4. 
+You can create a vog_config.json file in your project to modify the behaviour of vog. 
+Default settings generates PSR2 compliant value objects. 
 
-```php
-<?php
-require('vendor/marvinwank/vog/ConfigOptions.php');
-use Vog\ConfigOptions;
-
-return [
-    'generatorOptions' => [
-        'target' => ConfigOptions::MODE_PSR2,
-        'phpVersion' => ConfigOptions::PHP_74,
-    ],
-];
+```json
+{
+    "generatorOptions": {
+        "target": "MODE_PSR2"
+    }
+}
 ```
 
 Future releases may introduce more options. 
-Currently `ConfigOptions::MODE_FPP` generates value objects that are fpp compatible. 
+Currently the other mode available is `MODE_FPP`, which generates value objects that are fpp compatible. The differences are:
+
+for a property defined as `"property_name": "string"` in your json file.
+
+Method Type | PSR2 | FPP | Notes
+--- | --- | --- | ---
+Getter | getPropertyName() | property_name() | 
```

Comment:
Please add argument type and name as well as return types for better indication of the differences on the first sight